### PR TITLE
Remove deprecated entrypoints

### PR DESCRIPTION
**Pull request by Sniezka1927**

### CODE REVIEW

File: src/contracts/storage/tickmap.rs [hunk 1]
```diff
@@ -25,6 +25,7 @@ impl Default for Tickmap {
     }
 }
 
+#[allow(dead_code)]
 pub fn get_max_chunk(tick_spacing: u16) -> u16 {
```
Comment (zielvna) [inline src/contracts/storage/tickmap.rs:29]: can it be removed?

File: src/contracts/collections/pool_keys.rs [hunk 2]
```diff
@@ -34,38 +30,25 @@ impl PoolKeys {
...
         let offset_with_size = offset.checked_add(size).unwrap();
 
-        let max = if offset_with_size > self.pool_keys_length {
+        let upper_bound = if offset_with_size > self.pool_keys_length {
             self.pool_keys_length
         } else {
             offset_with_size
         };
 
+        let max = if upper_bound.checked_sub(offset).unwrap() > MAX_POOL_KEYS_RETURNED {
```
Comment (zielvna) [inline src/contracts/collections/pool_keys.rs:46]: same thing with `get_all` function in positions collection would be helpful

File: src/contracts/storage/pool_key.rs [hunk 3]
```diff
@@ -5,6 +5,8 @@ use crate::contracts::FeeTier;
 use crate::contracts::InvariantError;
 use crate::math::percentage::Percentage;
 
+pub const MAX_POOL_KEYS_RETURNED: u16 = 220;
```
Comment (zielvna) [inline src/contracts/storage/pool_key.rs:8]: maybe something more dynamic would be better: `MAX_RESULT / POOL_KEY_SIZE` (like we have it in wasm)

File: src/contracts/collections/pool_keys.rs (hunk 2 above)
Comment (wojciech-cichocki) [reply src/contracts/collections/pool_keys.rs:46]: Yes, we should add this to the `get_all` function in the `positions` collection as well

File: src/contracts/storage/tickmap.rs (hunk 1 above)
Comment (wojciech-cichocki) [reply src/contracts/storage/tickmap.rs:29]: We should keep this function as it's used in e2e tests. `#[allow(dead_code)]` is needed to remove the warning during the contract build

File: src/contracts/storage/pool_key.rs (hunk 3 above)
Comment (wojciech-cichocki) [reply src/contracts/storage/pool_key.rs:8]: definitely

**Review by zielvna — APPROVED** [review]

**Review by wojciech-cichocki — APPROVED** [review]

**Pull request merged**